web/profiles/custom/os2loop/themes/os2loop_theme/assets/app.scss

```diff
@@ -1,23 +1,58 @@
-@import 'bootstrap-icons/font/bootstrap-icons.css';
-$primary-color: #41ab51;
-$secondary-color: #f2f2f2;
+@import "bootstrap-icons/font/bootstrap-icons.css";
+@import "colors";
 
-.bg-secondary-loop {
-    background-color: $secondary-color;
+body {
+  font-family: "Montserrat" !important;
```

Comment:
Why `!important`?

It will probably be better to set Bootstrap's font variables (cf. https://github.com/twbs/bootstrap/blob/v4.6.0/scss/_variables.scss#L277-L279).